data/json/monsters/insect_spider.json

```diff
@@ -2969,5 +2969,379 @@
+    "copy-from": "mon_butterfly_giant",
+    "proportional": { "hp": 3, "speed": 1.1, "morale": 1.1, "melee_dice_sides": 1.1 },
+    "symbol": "L",
+    "color": "yellow",
+    "volume": "120 L",
+    "weight": "75 kg"
+  },
+  {
+    "id": "mon_moth",
+    "type": "MONSTER",
+    "name": { "str": "mutated moth" },
+    "description": "An oversized moth with its typical wood-like motifs.  It is as peaceful as the mutated butterflies found during the day.",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "default_faction": "insect",
+    "material": [ "iflesh" ],
+    "symbol": "l",
+    "color": "light_gray",
+    "volume": "15 L",
+    "weight": "20 kg",
+    "hp": 25,
+    "speed": 95,
+    "vision_night": 15,
+    "dodge": 2,
+    "harvest": "arachnid_firefly",
+    "reproduction": { "baby_egg": "egg_butterfly", "baby_count": 1, "baby_timer": 15 },
+    "upgrades": { "half_life": 14, "into": "mon_moth_giant" },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "fear_triggers": [ "HURT", "FIRE" ],
+    "flags": [ "SEES", "HEARS", "FLIES", "STUMBLES" ]
+  },
+  {
+    "id": "mon_moth_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant moth" },
+    "description": "A mutated moth larger than a human.  It silently flies with its huge wings after dusk looking for nocturnal flowers to feed upon.",
+    "bodytype": "flying insect",
+    "species": [ "INSECT_FLYING" ],
+    "default_faction": "insect",
+    "material": [ "iflesh" ],
+    "symbol": "L",
+    "color": "light_gray",
+    "volume": "70 L",
+    "weight": "40 kg",
+    "hp": 50,
+    "speed": 95,
+    "vision_night": 15,
+    "dodge": 2,
+    "harvest": "arachnid_firefly",
+    "biosignature": { "biosig_item": "insect_dust", "biosig_timer": 3 },
+    "reproduction": { "baby_egg": "egg_butterfly", "baby_count": 1, "baby_timer": 15 },
+    "baby_flags": [ "SPRING", "SUMMER" ],
+    "fear_triggers": [ "HURT", "FIRE" ],
+    "flags": [ "SEES", "HEARS", "FLIES", "STUMBLES" ]
+  },
+  {
+    "id": "mon_moth_titan",
+    "type": "MONSTER",
+    "name": { "str": "lunar moth" },
+    "description": "A majestic moth painted in pale colors.  This giant of the night only comes out after the moon is out, leaving glittering dust.",
+    "copy-from": "mon_moth_giant",
+    "proportional": { "hp": 3, "speed": 1.1, "morale": 1.1, "melee_dice_sides": 1.1 },
+    "symbol": "L",
+    "color": "cyan",
+    "volume": "120 L",
+    "weight": "75 kg"
+  },
+  {
+    "id": "mon_caterpillar",
+    "type": "MONSTER",
+    "name": { "str": "caterpillar" },
+    "description": "Caterpillars are the larval stage of butterflies and moths.  This one seems newly hatched from an egg but even now it is large as a cat.",
+    "default_faction": "ant",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "5 L",
+    "weight": "6 kg",
+    "hp": 10,
+    "material": [ "iflesh" ],
+    "symbol": "c",
+    "color": "green",
+    "speed": 50,
```

Comment:
I'd go even lower. Caterpillars are slow as hell, and half of walking speed feels like a stretch.

Reply:
reduced to 30